xtask/src/codegen/feature_docs.rs

```diff
@@ -8,7 +8,11 @@ use crate::{
     util::list_rust_files,
 };
 
-pub(crate) fn generate(_check: bool) {
+pub(crate) fn generate(check: bool) {
+    if check {
+        return;
+    }
```

Comment:
likewise `Feature::collect().unwrap()` should at least run with check enabled as it does some assertions